playbook/roles/filebeat/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Add Elasticsearch GPG key.
+  rpm_key:
+    key: https://packages.elastic.co/GPG-KEY-elasticsearch
+    state: present
+
+- name: Add Filebeat repository.
+  template:
+    src: beats.repo.j2
+    dest: /etc/yum.repos.d/beats.repo
+    mode: 0644
+
+- name: Install Filebeat.
+  package: name=filebeat state=present
+
+- name: Read hostname
```

Comment:
This should be available as `ansible_hostname` already

Reply:
When tested with ansible_hostname but it gives only short hostname (e.g. stage-web1 and not stage-web1.client-project.de-fra1.upc.wunder.io)